src/illuminator/models/Eboiler/eboiler_mosaik.py

```diff
@@ -25,7 +25,26 @@
 
 
 class eboilerSim(mosaik_api.Simulator):
-    def __init__(self):
+    def __init__(self) -> None:
+        """
+        Inherits the Mosaik API Simulator class and is used for python based simulations.
+        For more information properly inheriting the Mosaik API Simulator class please read their given documentation.
+
+        ...
+
+        Attributes
+        ----------
+        self.meta : dict
+            Contains metadata of the control sim such as type, models, parameters, attributes, etc.. Created via controlSim's parent class.
```

Comment:
Some of the doc strings are too long. Flake8 can be use to check where that is the case, and [black](https://pypi.org/project/black/) for automatic formatting. However, having doc stings is already a nice improvement, we can leave the formatting as a recommendation for the researches.